### PR TITLE
Refactor.

### PR DESCRIPTION
**Pull request by piohei**
## PR Checklist

- [ ] Added Tests
- [ ] Added Documentation

### CODE REVIEW

File: src/database/mod.rs [hunk 1]
```diff
@@ -29,6 +30,8 @@ pub struct Database {
     pub pool: Pool<Postgres>,
 }
 
+pub type SharedDatabase = Arc<Database>;
```
Comment (Dzejkop) [inline src/database/mod.rs]: `Arc<Database>` is actually fewer characters than `SharedDatabase` and obfuscates less

File: src/lib.rs [hunk 2]
```diff
@@ -7,6 +7,8 @@ pub mod config;
 mod contracts;
 mod database;
 mod ethereum;
+
+mod identity;
```
Comment (Dzejkop) [inline src/lib.rs:11]: Please rename `identity.rs` to `identity/mod.rs` we've decided recently to use mod named modules

File: src/identity_tree/initializer.rs [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+    ) -> Self {
+        Self {
+            database,
+            identity_manager,
+            config,
+        }
+    }
+
+    /// Initializes the tree state. This should only ever be called once.
+    /// Attempts to call this method more than once will result in a panic.
```
Comment (Dzejkop) [inline src/identity_tree/initializer.rs:37]: This is no longer true, right?

File: src/lib.rs (hunk 2 above)
Comment (gswirski) [reply src/lib.rs:11]: ditto `identity_tree` and `server`. as soon as a module has submodules, let's move it into directory for consistency

File: src/identity_tree/initializer.rs (hunk 3 above)
Comment (piohei) [reply src/identity_tree/initializer.rs:37]: Yup, you're right. Thanks!

**Pull request closed without merging**